webdriver/tests/bidi/browsing_context/capture_screenshot/origin.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from tests.support.image import png_dimensions
+
+from . import get_physical_viewport_dimensions, get_physical_document_dimensions
+
+
+@pytest.mark.asyncio
+async def test_origin(bidi_session, top_context, inline):
```

Comment:
@jrandolf it would be great if you could parametrize this test so that both settings can be tested individually and an early failure of `document` as origin will not prevent a check for `viewport`.  Ma

Reply:
We test the document screenshot by comparing it with the viewport screenshot.